doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+
+  If a webview is not set as active, it will not display any content.
+*/
+
+/*!
+  \qmlproperty bool WebView::loaded
+  \brief Whether the webview content has finished loading successfully
+*/
+
+/*!
+  \qmlproperty Qt::ScreenOrientation WebView::orientation
+  \brief The orientation of the webview
+
+  Ordinarily, the value of this property will follow the orientation of the
+  (Silica) Page in which the webview is shown.
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportWidth
+  \brief The width of the webview viewport
+
+  Note that this will usually be the same as the width of the webview, but
+  in some cases clients may wish to set the viewport width explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
+  \sa contentRect
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportHeight
+  \brief The height of the webview viewport
+
+  Note that this will usually be the same as the height of the webview, but
+  in some cases clients may wish to set the viewport height explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
+  \sa contentRect
+*/
+
+/*!
+  \internal
+  \qmlproperty QObject* WebView::child
+  \brief The webview object
+
+  // TODO: why does this property exist?  Can we remove it?
+  // It was added in b900d08cf8a98c4837562b7b63c78b6ac0600548
+  // But seems mostly useless, as it just returns "this"...
+
+  // If it is required, should we rename this property to _child ?
+*/
+
+/*!
+  \qmlmethod void WebView::setInputMethodHints(Qt.InputMethodHints hints)
+  \brief Allows setting input method hints
+
+  This affects the way the web view interprets input method events
+  (for example, enforcing formatted-numbers-only or dialable-characters-only
+  requirements as specified by the hints).
+*/
+
+// ---------------------------------------------------------- from Q_MOZ_VIEW:
+
+/*!
+  \qmlproperty QUrl WebView::url
+  \brief The URL being loaded and displayed by the webview
+
+  Clients can set this property to navigate to a new location.
+  If the webview is redirected, the value of this property will automatically
+  be updated.
+*/
+
+/*!
+  \qmlproperty QString WebView::title
+  \readonly
+  \brief The page title of the currently loaded location
+*/
+
+/*!
+  \qmlproperty bool WebView::canGoBack
+  \readonly
+  \brief Whether the "back" navigation action is available
+
+  If the user has previously navigated to a different page via setting the
+  \l{url} property, then back navigation will be available.
+
+  \sa goBack()
+*/
+
+/*!
+  \qmlproperty bool WebView::canGoForward
+  \readonly
+  \brief Whether the "forward" navigation action is available
+
+  If the user has previously navigated back (via \l{goBack()}) without later
```

Comment:
```suggestion
  If the user has navigated back (via \l{goBack()}) without later
```

Reply:
Actually not sure if previously makes more sense here as there's previously - later.

Reply:
done